test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java

```diff
@@ -553,26 +563,30 @@ private static void startTestApplicationContainer() throws Exception {
 
         final ContainerInfo containerInfo = new ContainerInfo(containerId, currentImageName);
         currentContainerInfo.set(containerInfo);
-        try {
-            String url = String.format("http://localhost:%s/", String.valueOf(appServerPort));
-            System.out.printf("Verifying appserver has started (%s)...%n", url);
+        if (currentImageName.startsWith("javase_")) {
```

Comment:
what's the rationale here?

Reply:
👍 Good point, I was thinking with Java SE app there would not be an app server to handle the url, but that's not the case.

Reply:
I forgot why 😭. There was a reason. I added comment to explain.